bin/internal/shared.bat

```diff
@@ -61,6 +61,13 @@ IF NOT EXIST "%cache_dir%" (
 GOTO :after_subroutine
 
 :subroutine
+  REM If present, run the bootstrap script first
+  SET bootstrap_path=%FLUTTER_ROOT%\bin\internal\bootstrap.bat
+  IF EXIST "%bootstrap_path%" (
+    ECHO Running bin/internal/bootstrap.bat... 1>&2
```

Comment:
Since these will run every time, I think the additional logs should be left out, it will make startup somewhat noisy